src/vs/base/browser/ui/tree/media/tree.css

```diff
@@ -38,6 +38,12 @@
 	height: 100%;
 }
 
+.monaco-tl-twistie + .monaco-tl-contents {
+	padding-left: 6px;
+	margin-left: -6px;
```

Comment:
We could remove `padding-right` in `.monaco-tl-twistie` instead of using margin-left here, but I'm worried that the twistie is also being used somewhere else.